app/src/main/res/layout/item_daily_word_list_home.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+
+    >
+    <data>
+        <variable
+            name="item"
+            type="com.vinh.dictionary_1.data.model.DailyWord"
+            />
+
+        <variable
+            name="dailyWordDrawable"
+            type="Integer"
+            />
+
+        <variable
+            name="itemListener"
+            type="com.vinh.dictionary_1.screen.BaseRecyclerViewAdapter.OnRecyclerViewItemClickListener"
+            />
+    </data>
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="@dimen/dp_180"
+        android:layout_marginEnd="@dimen/dp_15"
+        android:layout_marginLeft="@dimen/dp_15"
+        android:layout_marginRight="@dimen/dp_15"
+        android:layout_marginStart="@dimen/dp_15"
+        android:layout_marginTop="@dimen/dp_10"
+        android:background="@drawable/shape_round_corner_home_daily_word"
+        android:onClick="@{(v) -> itemListener.onItemRecyclerViewClick(v, item)}"
+        android:orientation="vertical"
+        >
+
+        <ImageView
+            android:layout_width="@dimen/dp_320"
+            android:layout_height="@dimen/dp_100"
+            android:layout_gravity="center_horizontal"
+            android:imageResource="@{dailyWordDrawable}"
+            />
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/dp_15"
+            android:layout_marginLeft="@dimen/dp_15"
+            android:layout_marginRight="@dimen/dp_15"
+            android:layout_marginStart="@dimen/dp_15"
+            >
+
+            <TextView
+                android:id="@+id/text_word"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:ellipsize="end"
+                android:maxLength="20"
+                android:maxLines="1"
+                android:text="@{item.word}"
+                android:textColor="@color/color_black"
+                android:textSize="@dimen/sp_17"
+                android:textStyle="bold"
+                />
+
+            <TextView
```

Comment:
WARNING: [RelativeOverlap] `@id/text_pronounce` can overlap `@id/text_word` if @{item.word}, @{item.pronounce} grow due to localized text expansion
 If relative layout has text or button items aligned to left and right sides they can overlap each other due to localized text expansion unless they have mutual constraints like `toEndOf`/`toStartOf`.